automod/engine/persist.go

```diff
@@ -57,8 +57,13 @@ func (eng *Engine) persistAccountModActions(c *AccountContext) error {
 	if err != nil {
 		return fmt.Errorf("circuit-breaking takedowns: %w", err)
 	}
+	// @TODO: do we want to check if the account is already escalated?
```

Comment:
yeah, we should check here. we should also maybe popular review start as part of account meta? and clear account meta when it changes.